account_invoice_merge/models/account_invoice.py

```diff
@@ -7,70 +7,65 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, models
-from odoo.osv.orm import browse_null, browse_record
 from odoo.tools import float_is_zero
 
 
 class AccountInvoice(models.Model):
```

Comment:
Class should be renamed to `AccountMove` and maybe that file renamed to `account_move.py` in a separate commit

Reply:
Done!